docparse/chunking_tutorial.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: 'Chunking Tutorial'
+description: 'Diving deep into each of the chunking strategies supported by Aryn DocParse'
+icon: 'brackets-curly'
```

Comment:
I'll suggest using a different icon for this from the other chunking page (and maybe from other pages as well)